tendrl-alerting.service

```diff
@@ -0,0 +1,13 @@
+[Unit]
```

Comment:
Please keep this inline with https://github.com/Tendrl/performance-monitoring/blob/master/tendrl-performance-monitoring.service